platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -460,24 +470,53 @@ public class SwirldsPlatform implements Platform {
 
         components.add(new IssMetrics(platformContext.getMetrics(), currentAddressBook));
 
+        // Manages the pipeline of signed states to be written to disk
+        final SignedStateFileManager signedStateFileManager = new SignedStateFileManager(
+                platformContext,
+                new SignedStateMetrics(platformContext.getMetrics()),
+                Time.getCurrent(),
+                actualMainClassName,
+                selfId,
+                swirldName);
+        // FUTURE WORK: at some point this should be part of the unified platform wiring
+        final WiringModel model = WiringModel.create(platformContext, Time.getCurrent());
+        final TaskScheduler<StateSavingResult> savedStateScheduler = model.schedulerBuilder("signed_state_file_manager")
+                .withType(TaskSchedulerType.SEQUENTIAL_THREAD)
+                .withUnhandledTaskCapacity(stateConfig.stateSavingQueueSize())
+                .withExternalBackPressure(false)
```

Comment:
This is default `false`, so you don't need to specify it. The reason why this method exists is because of Oleg's request that we support schedulers with external back pressure. If you do have external back pressure, you need some way of telling the wiring model that you are doing this, otherwise it may not properly identify cyclic back pressure. When you call `.withUnhandledTaskCapacity(stateConfig.stateSavingQueueSize())`, the model understands that this is a back pressure mechanism, and will correctly identify cyclic back pressure involving this component.

Reply:
got it, removed